README.md

```diff
@@ -76,20 +76,24 @@ If you have not yet cloned the Harmony repository, run
 $ git clone https://github.com/nasa/harmony.git
 ```
 
-Ensure node is available and is the correct version, 12.x.x
+Ensure node is available and is the correct version, 12.x.y, where "x" >= 14.
```

Comment:
maybe should add "y" >= 1?

Reply:
yeah, I really couldn't think of a concise way to say "^12.14.1".  Open to suggestions.  Technically y could be 0 if x > 14.